app/helpers/avatar_helper.rb

```diff
@@ -102,7 +86,8 @@ def with_default_avatar_options(user, options, &block)
     end
 
     if Setting.gravatar_enabled? && (email = extract_email_address(user)).present?
-      block.call(email.to_s.downcase, options)
+      block.call email.to_s.downcase,
+                 options.merge(gravatar: default_gravatar_options)
```

Comment:
Wouldn't this mean options get overridden by the default options even if you give another value for `default_image` for instance?